content/learn/getting-started.es.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Primeros pasos
+mobile_menu_title: "Primeros pasos"
+toc: true
+---
+
+## Lanzamientos etiquetados o compilación nocturna?
+Zig no ha alcanzado la versión 1.0 aún y el ciclo de liberación actual depende de la liberación de LLVM, lo cual ocurre cada ~6 meses.
+En terminos prácticos, **los lanzamientos de Zig tienden a ser muy separados entre si y ocasionalmente llegan a perder vigencia debido a la velocidad en el desarrollo del lenguaje.**.
+
+No hay problema con evaluar Zig utilizando alguna de las versiones etiquetadas, pero si deciedes que quieres continuar programando con Zig mas a fondo, **te recomendamos actualizar a las compilaciones nocturnas**, ya que de esa manera será mas fácil obtener ayuda: la mayoría de los miembros de la comunidad y sitios como [ziglearn.org](https://ziglearn.org) siguen la rama "master" del repositorio por las razones expuestas arriba.
+
+La buena noticia es que es muy fácil de moverse de una versión de Zig a otra, o incluso tener multimples versiones presentes en el sistema al mismo tiempo: las versiones de Zig son un simple folder con todo lo necesario, el cual puede ser colocado en cualquier ubicación del sistema.
+
+## Instalación de Zig
+### Descarga directa
+Esta es la manera mas simple de obtener Zig: descarga un paquete de Zig correspondiente a tu plataforma desde la [página de descargas](/es/download), extrae el contenido en el directorio de tu elección y añade la rua al `PATH` para poder ejecutar `zig` desde cualquier ubicación.
+
+#### Configuración de la variable de entorno PATH en Windows
+Para configurar la ruta de Zig en el `PATH` the Windows ejecuta **uno** de los siguientes fragmentos de código en una terminal Powershell. Elige si quieres aplicar el cambio para todos los usuarios del sistema (lo cual requiere ejecutar Powershell con derechos de administrador) o solo para el usuario actual, y **asegurate de ajustar el fragmento de código para que apunte a la ubicación de tu copia de Zig**. El `;` antes de `C:` no es un error.
+
+Para todos los usuarios (Powershell ejecutado como **administrador**):
+```
+[Environment]::SetEnvironmentVariable(
+   "Path",
+   [Environment]::GetEnvironmentVariable("Path", "Machine") + ";C:\tu-ruta\zig-windows-x86_64-tu-versión",
+   "Machine"
+)
+```
+
+Para el usuario actual (Powershell):
+```
+[Environment]::SetEnvironmentVariable(
+   "Path",
+   [Environment]::GetEnvironmentVariable("Path", "User") + ";C:\tu-ruta\zig-windows-x86_64-tu-versión",
+   "User"
+)
+```
+Despues de esto, reinicia la terminal de Powershell.
+
+#### Configuración de la variable de entorno PATH Linux, macOS, BSD
+Añade la ubicación del ejecutable zig a la variable de ambiente PATH.
+
+Generalmente, esto se agregando una linea de 'export' a tu script de inicio del shell (`.profile`, `.zshrc`, ...)
+```bash
+export PATH=$PATH:~/ruta/a/zig
+```
+Despues de esto, reinicia tu shell o refresca con el comando `source`
```

Comment:
`actualizar` might be better than `refrescar` here

Reply:
agree